packages/addons/service/emby/changelog.txt

```diff
@@ -1,4 +1,11 @@
+<<<<<<< HEAD
 114
+=======
+115:
+- Update to version 3.2.26.0
+
+114:
+>>>>>>> 35c307893... Emby: update to 3.2.26.0
```

Comment:
Thanks, I'm still getting the hang of the git console but can already see it is much more powerful than the Web UI. I will pop in on IRC next time I am struggling.